### ISSUE TITLE
No documentation on source of WEBSTORE_OAUTH2_FILE and WEBSTORE_SECRETS_FILE

### ISSUE THREAD

**Issue by eric-wieser** [jonnor/chrome-webstore-deploy#1]
Not immediately obvious how exactly these files are obtained

**Comment by ssbarnea** [comment]
I totally agree. Some minimal documentation regarding on how to get them it would more than welcome. 

So far I am looking at the Credentials screen from Google API and I do not see how I can get them.

**Comment by brothaakhee** [comment]
I'm having this same question. After some digging I found https://developers.google.com/api-client-library/python/guide/aaa_client_secrets and it talks about storing secrets in a json file

**Comment by dluong-pi** [comment]
After my research, found out how to obtain these files:
- `oauth2.dat` is use to store credentials after you have authorize succeeded once. You could create it by run Terminal `$ touch oauth2.dat`
- `secrets.json` is provide by [Google Developers Console | https://console.developers.google.com/] by create **OAuth client ID** with Application type: **Other**. Then you could download json file in edit page.
- More information about Chrome Web Store API https://developer.chrome.com/webstore/using_webstore_api
